[2020-07-15 14:50:04] local.ERROR: No Database connection yet in VoyagerServiceProvider loadAuth()  
[2020-07-15 14:52:04] local.ERROR: No Database connection yet in VoyagerServiceProvider loadAuth()  
